integreat_cms/cms/forms/pages/page_form.py

```diff
@@ -196,3 +198,19 @@ def get_publisher_queryset(self):
                 self.instance.publishers.all()
             )
         return users_without_permissions
+
+    def save(self, commit=True):
+        """
+        This method extends the default ``save()``-method of the base :class:`~django.forms.ModelForm` to flush
+        the cache after commiting.
+
+        :param commit: Whether or not the changes should be written to the database
+        :type commit: bool
+
+        :return: The saved page translation object
+        :rtype: ~integreat_cms.cms.models.pages.page_translation.PageTranslation
+        """
+        # Save CustomModelForm and flush Cache
+        result = super().save(commit=commit)
+        invalidate_model(Page)
+        return result
```

Comment:
```suggestion
```
This also happens anyway because the `move()` method is called whenever a page is saved. And I guess it's not required to invalidate the `PageTranslation` model because it has a foreign key relation to the page model and thus invalidated anyway.